scripts/create_gitlab_snapshot.py

```diff
@@ -49,22 +49,22 @@ def main():
 
 
 def gitlab_volume_info() -> Mapping[str, JSON]:
-    return one(aws.ec2.describe_volumes(Filters=gitlab_filter)['Volumes'])
+    return only(aws.ec2.describe_volumes(Filters=gitlab_filter)['Volumes'])
```

Comment:
I don't think so. There is no conditional at the call site to handle a `None` return value. Also, the return type hint doesn't match.